.github/actions/docker-build-and-push-openadkit/action.yaml

```diff
@@ -0,0 +1,179 @@
+name: docker-build-and-push-openadkit
+description: ""
+
+inputs:
+  bake-target:
+    description: ""
+    required: true
+  bake-images:
+    description: ""
+    required: false
+  build-args:
+    description: ""
+    required: false
+  tag-prefix:
+    description: ""
+    required: false
+  tag-suffix:
+    description: ""
+    required: false
+  tag-arch:
+    description: ""
+    required: false
+  allow-push:
+    description: ""
+    default: "true"
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Install jq
+      run: |
+        sudo apt-get -y update
+        sudo apt-get -y install jq
```

Comment:
What are we using jq for?

Reply:
I don't think it's being used with any other actions, so we can remove it.